pennylane/measurements.py

```diff
@@ -587,3 +593,115 @@ def circuit():
     """
     # pylint: disable=protected-access
     return MeasurementProcess(State, wires=qml.wires.Wires(wires))
+
+
+T = TypeVar("T")
+
+
+class MeasurementValueError(ValueError):
+    """Error raised when an unknown measurement value is being used."""
+
+
+class MeasurementValue(Generic[T]):
+    """A class representing unknown measurement outcomes in the qubit model.
+
+    By default, measurements in the computational basis are assumed.
```

Comment:
```suggestion
    Measurements on a single qubit in the computational basis are assumed.
```
Not sure, but do we have the capability to be "not default"?

Reply:
Under the hood, the zero and one case outcomes can be passed to `MeasurementValue`. Having said that, this is not accessible to the user atm.